web/src/components/Toolbar.tsx

```diff
@@ -36,18 +35,28 @@ const Headline = styled.h5`
   font-size: 90%;
 `
 
-type PropsType = {
+type ToolbarProps = {
   className?: string
   children?: ReactNode
   viewportSmall: boolean
-  t: TFunction
+  iconDirection?: 'row' | 'column'
 }
 
-const Toolbar = ({ children, className, t, viewportSmall }: PropsType): ReactElement => (
-  <ToolbarContainer className={className}>
-    {viewportSmall && <Headline>{t('isThisSiteUseful')}</Headline>}
-    {children}
-  </ToolbarContainer>
-)
+const Toolbar: React.FC<ToolbarProps> = ({
```

Comment:
refactored to functional component

Reply:
Was a functional component before already? :D

Reply:
oh sorry your are right :D just removed HOC